gax/test/unit/regapic.ts

```diff
@@ -569,4 +583,86 @@ describe('REGAPIC', () => {
       }, /* catch: */ done);
     });
   });
+
+  describe('should support json minification', () => {
+    it('should send prettyPrint=0 when json minification is requested', done => {
+      const requestObject = {name: 'shelves/shelf-name'};
+      const responseObject = {
+        name: 'shelf-name',
+        theme: 'shelf-theme',
+        type: 100, // unknown enum value
+      };
+      const spy = sinon.spy(transcoding, 'transcode');
+      // incomplete types for nodeFetch, so...
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      sinon.stub(nodeFetch, 'Promise' as any).returns(
+        Promise.resolve({
+          ok: true,
+          arrayBuffer: () => {
+            return Promise.resolve(Buffer.from(JSON.stringify(responseObject)));
+          },
+        })
+      );
+
+      gaxGrpcMinifyJson
+        .createStub(libraryService, stubOptions)
+        .then(libStub => {
+          libStub.getShelf(requestObject, {}, {}, (err?: {}, result?: {}) => {
+            assert.match(
+              spy.getCall(0).returnValue?.queryString ?? '',
```

Comment:
In general I don't think we should be unsure of what we're matching (I don't think we should be using the nullish coalescing operator in tests)